cmd/scripts/getLastUpdateTime.sh

```diff
@@ -63,6 +63,10 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+if [ "$cluster" == "local-cluster" ]; then
```

Comment:
How about this:
1. get the actual local-cluster name
```
% oc get managedclusters  -l local-cluster=true
```
2. if $cluster == the local cluster name,  change the appName  as implemented

Reply:
Sounds good. I didn't know local-cluster could be a different name.